backend/routes/adding complete question

```diff
@@ -0,0 +1,24 @@
+Python Script to Insert data to databases:
```

Comment:
Wrong folder (routes). Move into scripts folder. Rename the filename (adding_complete_questions_to_database)